frontend/chains/agoric.json

```diff
@@ -21,8 +21,8 @@
     "config": {
       "chain_id": "agoric-3",
       "chain_name": "Agoric",
-      "rest": "https://api.resolute.vitwit.com/agoric_api/",
-      "rpc": "https://api.resolute.vitwit.com/agoric_rpc/",
+      "rest": "https://agoric-api.polkachu.com",
```

Comment:
Why are we using another provider? Is our endpoint not working?

Reply:
Yes our endpoint is returning connection refused error